### PR TITLE
add docs for builtin aggregation functions

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

File: sql/functions-and-operators/aggregate-group-by-functions.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Aggregate (GROUP BY) Functions
+
+## Aggregate (GROUP BY) Function Descriptions
+
+see: [https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html](https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html)
```
Comment (queenypingcap) [inline sql/functions-and-operators/aggregate-group-by-functions.md]: see: [https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html](https://dev.mysql.com/doc/refman/5.7/en/group-by-functions.html) ->
This section describes the supported MySQL group (aggregate) functions in TiDB.

File: sql/functions-and-operators/aggregate-group-by-functions.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+| `STD()`                                                                                                       | not supported                                     |
+| `STDDEV()`                                                                                                    | not supported                                     |
+| `STDDEV_POP()`                                                                                                | not supported                                     |
+| `STDDEV_SAMP()`                                                                                               | not supported                                     |
+| `VARIANCE()`                                                                                                  | not supported                                     |
+| `VAR_POP()`                                                                                                   | not supported                                     |
+| `VAR_SAMP()`                                                                                                  | not supported                                     |
+
+
+## GROUP BY Modifiers
```
Comment (queenypingcap) [inline sql/functions-and-operators/aggregate-group-by-functions.md]: Please confirm whether the content is exactly the same as MySQL first.

File: sql/functions-and-operators/aggregate-group-by-functions.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+see: [https://dev.mysql.com/doc/refman/5.7/en/group-by-modifiers.html](https://dev.mysql.com/doc/refman/5.7/en/group-by-modifiers.html)
+
+
+## MySQL Handling of GROUP BY
```
Comment (queenypingcap) [inline sql/functions-and-operators/aggregate-group-by-functions.md]: Ditto.

File: sql/functions-and-operators/aggregate-group-by-functions.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+see: [https://dev.mysql.com/doc/refman/5.7/en/group-by-handling.html](https://dev.mysql.com/doc/refman/5.7/en/group-by-handling.html)
+
+## Detection of Functional Dependence
```
Comment (queenypingcap) [inline sql/functions-and-operators/aggregate-group-by-functions.md]: Ditto.

**Comment by zz-jason** [comment]
@queenypingcap PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**